src/Sarif.Multitool/Rules/SarifValidationSkimmerBase.cs

```diff
@@ -22,12 +22,7 @@ public abstract class SarifValidationSkimmerBase : Skimmer<SarifValidationContex
 
         public override Uri HelpUri => _defaultHelpUri;
 
-        private readonly MultiformatMessageString _emptyHelpMessage = new MultiformatMessageString
-        {
-            Text = string.Empty
-        };
-
-        public override MultiformatMessageString Help => _emptyHelpMessage;
+        public override MultiformatMessageString Help => null;
```

Comment:
>null [](start = 57, length = 4)

I've made the same change in my pending PR #1570.

Reply:
ack

---
In reply to: [303171587](https://github.com/microsoft/sarif-sdk/pull/1574#discussion_r303171587) [](ancestors = 303171587)